http_service/tests/test_bug_classification.py

```diff
@@ -159,6 +159,41 @@ def do_request():
     }
 
 
+def test_for_missing_bugs(client, responses):
+    existed_bugs_ids = [1602463, 1619699]
+    missing_bugs_ids = [1598744, 1615281, 1566486]
```

Comment:
`existed_bugs_ids` --> `existed_bug_ids`
`missing_bugs_ids` --> `missing_bug_ids`

Reply:
I don't understand this, sir

Reply:
well noted